.werft/platform-delete-preview-environments-cron.ts

```diff
@@ -411,58 +216,69 @@ async function determineStalePreviewEnvironments(options: {
 
 async function removePreviewEnvironment(previewEnvironment: PreviewEnvironment) {
     const sliceID = `Deleting preview ${previewEnvironment.name}`;
-    werft.log(sliceID, `Starting deletion of all resources related to ${previewEnvironment.name}`);
+    werft.log(sliceID, `Triggering job to delete ${previewEnvironment.name}. DRY_RUN=${DRY_RUN}`);
     try {
-        // We're running these promises sequentially to make it easier to read the log output.
-        await previewEnvironment.removeDNSRecords(sliceID);
-        await previewEnvironment.delete();
+        exec(
+            `sudo chown -R gitpod:gitpod /workspace && git config --global user.name roboquat && git config --global user.email roboquat@gitpod.io`,
```

Comment:
Do we need this on every execution of removePreviewEnvironment? Could we move this to to the yaml file - that's where we have this "init" code in other jobs ☺️